app/src/main/java/org/cru/godtools/fragment/ToolDetailsFragment.java

```diff
@@ -280,24 +288,23 @@ private void updateViews() {
             mTitle.setText(ModelUtils.getTranslationName(getContext(), mLatestTranslation, mTool));
         }
         bindShares(mShares, mTool);
-        if (mDescription != null) {
-            mDescription.setText(ModelUtils.getTranslationDescription(getContext(), mLatestTranslation, mTool));
-        }
-        if (mLanguagesHeader != null) {
-            final int count = mLanguages.size();
-            mLanguagesHeader.setText(mLanguagesHeader.getResources()
-                                             .getQuantityString(R.plurals.label_tools_languages, count, count));
-        }
-        if (mLanguagesView != null) {
-            mLanguagesView.setVisibility(mLanguages.isEmpty() ? View.GONE : View.VISIBLE);
-            mLanguagesView.setText(Stream.of(mLanguages)
-                                           .map(l -> LocaleUtils.getDisplayName(l, mLanguagesView.getContext(), null,
-                                                                                null))
-                                           .withoutNulls()
-                                           .sorted(String.CASE_INSENSITIVE_ORDER)
-                                           .reduce((l1, l2) -> l1 + ", " + l2)
-                                           .orElse(""));
+
+        final int count = mLanguages.size();
+        if (mViewPager != null) {
+            ToolsDetailAdapter adapter = new ToolsDetailAdapter();
```

Comment:
you didn't include the adapter and it should probably be named `ToolDetailsAdapter`

Reply:
@frett not sure what you mean by I didn't include the adapter?

Reply:
I hadn't noticed it was an inner class, I was looking for a separate `ToolDetailsAdapter` source file.